### PR TITLE
Fixes poorly-styled captions in spec

### PR DESCRIPTION
**Pull request by meiqimichelle**
For #7.

This PR creates semantically styled markup/CSS for figure captions in our spec.

**Before**, our ASCII art illustrations looked like this:
Screenshot 2020-01-27 20 40 13

There were a number of readability issues with this display, including:
- The art was being colored as if it were code, which wasn't adding any clarity (and sometimes making the art harder to understand)
- The captions were floating waaaay below the figure itself, and were displayed using the default text styling.
- On the markup side, nothing semantically connected the caption to the figure, which is not good for accessibility.

**With this PR**, our illustrations and captions look like this:
Screenshot 2020-01-27 21 13 43

- The grey code box and text coloring is gone, leaving a nice strong black art that blends well with the rest of the text
- The captions are snugged up along the bottom of the figures, and are smaller and grayer than normal text.
- The markup is correct to connect the figure to its caption.

### CODE REVIEW

File: content/docs/spec.md [hunk 1]
```diff
@@ -120,13 +120,14 @@ The server MUST wait until C0 has been received before sending S0 and S1, and MA
...
 
 The C0 and S0 packets are a single octet, treated as a single 8-bit integer field:
-```
+
+<figure><pre>
  0 1 2 3 4 5 6 7
 +-+-+-+-+-+-+-+-+
 |    version    |
 +-+-+-+-+-+-+-+-+
  C0 and S0 bits
```
Comment (ultrasaurus) [inline content/docs/spec.md]: prolly want `C0 and S0 bits` to be figcaption?

File: content/docs/spec.md [hunk 2]
```diff
@@ -151,7 +153,8 @@ The C1 and S1 packets are 1536 octets long, consisting of the following fields:
 |                             ...                               |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
                         C1 and S1 bits
```
Comment (ultrasaurus) [inline content/docs/spec.md]: should `C1 and S1 bits` be <figcaption> for consistency?

**Review by ultrasaurus — APPROVED** [review]
I like this solution!  Just left two minor comments where I think you might have wanted a figcaption.  Fine if you think those are better as part of the figure, just let me know.

File: content/docs/spec.md (hunk 1 above)
Comment (meiqimichelle) [reply content/docs/spec.md]: I wasn't sure if those were meant to be captions or not when I read the original PDF. Looking again, I think you're right -- they were meant to be captions. I'll update and push a fix. Thx for catching!

File: content/docs/spec.md (hunk 2 above)
Comment (meiqimichelle) [reply content/docs/spec.md]: Yup, agree. (Same issue as https://github.com/veriskope/rtmp-website/pull/18#discussion_r371599109)

**Pull request merged**